src/triagebot/transfer.md

```diff
@@ -0,0 +1,36 @@
+# Issue Transfer
+
+The `transfer` command allows you to transfer a GitHub issue from one repository to another.
+
+## Usage
+
+To transfer an issue to another repository, enter a comment with the form:
+
+`@rustbot transfer` *repository-name*
```

Comment:
```suggestion
`@rustbot transfer <repository-name>`
```

(style) more legible imo and a pretty common way to denote placeholders within “commands” across domains